webview/src/experiments/components/table/TableHeader.tsx

```diff
@@ -130,7 +130,7 @@ const getIconMenuItems = (
   }
 ]
 
-const FirstTableHeaderCellWrapper: React.FC<{
+const WithExpColumnNeedsShadowUpdates: React.FC<{
```

Comment:
I tried to make the naming more clear but this sounds weird to me as well... Open to other suggestions 😅